tracking/translations_parser/publishers.py

```diff
@@ -99,16 +99,25 @@ def open(self, parser) -> None:
         config.update(self.extra_kwargs.pop("config", {}))
 
         try:
-            # Check if a W&B run already exists with this name
             project = next(filter(lambda p: p.name == self.project, wandb.Api().projects()), None)
+            # Check if a W&B run already exists with this name
+            existing_runs = []
             if project and (name := self.extra_kwargs.get("name")):
                 existing_runs = list(
                     wandb.Api().runs(
                         self.project,
                         filters={"display_name": name, "group": self.extra_kwargs.get("group")},
                     )
                 )
-                if len(existing_runs) > 0:
+            if len(existing_runs) > 0:
+                if os.environ.get("RUN_ID", 0) > 0:
+                    logger.info(
+                        "Training has been resumed from an earlier run, data will be overwritten on W&B."
+                    )
+                    for run in existing_runs:
+                        logger.warning(f"Deleting existing run {run.display_name}.")
+                        run.delete()
```

Comment:
@vrigal it's not supposed to delete the data from the previous run. It should continue tracking like the restart never happened.

Reply:
See https://docs.wandb.ai/guides/runs/resuming#resuming-guidance